src/Apps/Artist/Routes/AuctionResults/Components/SortSelect.tsx

```diff
@@ -23,6 +23,7 @@ export const SortSelect = () => {
     <Subscribe to={[AuctionResultsState]}>
       {(filters: AuctionResultsState) => (
         <SelectSmall
+          title="Sort"
```

Comment:
Does this still look good on mobile? Part of the reason it was removed was to make consistent with responsive. @oxaudo would know more.